src/postInstall.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
```

Comment:
This script looks very similar to [the script that exists in the expo plugin](https://github.com/customerio/customerio-expo-plugin/pull/8/files). 

Instead of copy/pasting the script, what about having the RN SDK run the script found inside of the expo plugin? 

```javascript
const postInstallScriptExpoPlugin = JSON.parse(require('customerio-expo-plugin')).scripts.postInstall;

os.exec(postInstallScriptExpoPlugin)
```

This code is a little pseudo code but it's just to communicate an idea.

Reply:
One thing to consider is that we might not want our `post-install` script to be called from outside of the module. It is meant to be `private` to the module it is added to.

Reply:
I am still not sure it's a good idea to have almost identical scripts in 2 separate repositories. 

For maintainability and preventing bugs, having a script exist in 1 place makes more sense to me.